ReactQt/runtime/src/reactuimanager.cpp

```diff
@@ -129,7 +129,7 @@ void ReactUIManager::setChildren
   //TODO: This is a simple implementation which fixes a broken example. It's not properly tested and may need revisiting
   QList<int> indices;
   foreach(int i, childrenTags) {
-    indices.append(1); //same Z order for all items
+    indices.append(0); //0 is a default z-order in qml
```

Comment:
@MaxRis , @vkjr.  When I was setting 0 Z order, the app title was drawn at the bottom (instead of top). Of course, that issue is not caused by Z order, just the same indices were used as indices in layout. 
will double check that.